t/dedupe.c

```diff
@@ -495,12 +569,16 @@ static void show_stat(uint64_t nextents, uint64_t nchunks, uint64_t ndupextents)
 	perc *= 100.0;
 	printf("Fio setting: dedupe_percentage=%u\n", (int) (perc + 0.50));
 
+
+	if (compression) {
+		uc_human = bytes_to_human_readable_unit(unique_capacity, &unit);
+		printf("Unique capacity %lu%s\n", uc_human, unit);
```

Comment:
Got a warning here:
```
t/dedupe.c:575:43: warning: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 2 has type ‘uint64_t’ {aka ‘long long unsigned int’} [-Wformat=]
  575 |                 printf("Unique capacity %lu%s\n", uc_human, unit);
      |                                         ~~^       ~~~~~~~~
      |                                           |       |
      |                                           |       uint64_t {aka long long unsigned int}
      |                                           long unsigned int
      |                                         %llu
```

The specifier for `uint64_t` may not always be (unsigned long). It is sometimes (unsigned long long).

So please cast this one to `unsigned long` like the other parts.
```suggestion
		printf("Unique capacity %lu%s\n", (unsigned long) uc_human, unit);
```

Reply:
fixed